<?xml version="1.0"?>
<launch>
    <arg name="carrier_name" default="raptor"/>
    <arg name="carrier_type" default="generic_quadplane"/>

    <group ns="$(arg carrier_name)">
        <node pkg="iarc_mission_nine" type="carrier_state_machine_node" name="carrier_state_machine" output="screen">
            <rosparam file="$(find iarc_mission_nine)/cfg/params.yaml" command="load"/>
            <remap from="odometry" to="ground_truth/odometry"/>
        </node>
    </group>
</launch>
